probe-rs/targets/esp32c6.yaml

```diff
@@ -13,6 +13,9 @@ variants:
       - name: main
         type: riscv
         core_access_options: !Riscv {}
+      - name: lp
+        type: riscv
+        core_access_options: !Riscv {}        
```

Comment:
How are these associated with the harts? If it's implicit, can we make this explicit by inlcuding the hart's ID in the Riscv struct?

Reply:
Agree - probably would be better to have the hart-id as core_access_options 🤔